website-profile.html

```diff
@@ -1 +1,54 @@
-<!DOCTYPE html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Lewis's Profile</title>
+<link rel="stylesheet" href="style.css">
+<body>
+
+<div class="top-banner">
```

Comment:
Semantic HTML is great for developer and machine readability as well as accessibility for those with impairments. 

```suggestion
<header class="top-banner">
```